### PR TITLE
Feature/userprofile

### PR DESCRIPTION
**Pull request by EvelynCSL**
Changes:
- profile page html
- css for profile page
- included mock data for user (as i am not too sure how to directly implement into the datastore)
- create user class
- create userServlet (to be completed)
- included async function on js to fetch data

Outstanding work:
- modify userServlet to get user data from mockdata file
- display past reviews
- display user's favourited business
- determine what variable user class needs

### CODE REVIEW

File: src/main/java/com/google/sps/servlets/userServlet.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+public class userServlet extends HttpServlet {
+
+    Gson gson = new Gson();
+    MockUserData mockUserData = new MockUserData();
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+        response.setContentType("text/html;");
+        response.getWriter().println("<h1>Hello world!</h1>");
```
Comment (dalisc) [inline src/main/java/com/google/sps/servlets/userServlet.java:20]: I think we want to print out the mock data, not Hello World!

File: src/main/webapp/assets/js/userprofile.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+async function getUserInformation() {
+    let response = await fetch('/userinformation');
+    let userinformation = await response.json();
+    return userinformation;
```
Comment (dalisc) [inline src/main/webapp/assets/js/userprofile.js:4]: I suppose this is just a dummy for now right? Will be returning something else or formatting the response gotten as the development goes along?

Comment (EvelynCSL) [reply src/main/webapp/assets/js/userprofile.js:4]: yeap this is a dummy for now. yeap i will modify this along the way.

File: src/main/java/com/google/sps/servlets/userServlet.java (hunk 1 above)
Comment (EvelynCSL) [reply src/main/java/com/google/sps/servlets/userServlet.java:20]: yeap i will modify this and send another pull request! :)

**Pull request merged**